packages/blocks/src/store/private-selectors.js

```diff
@@ -93,6 +130,36 @@ function filterElementBlockSupports( blockSupports, name, element ) {
 	} );
 }
 
+function getStableBlockSupports( blockType ) {
```

Comment:
TODO: Add a comment explaining what this does (treats experimental typography keys as non-experimental)